frontend/src/metabase/ui/components/inputs/MultiSelect/MultiSelect.stories.js

```diff
@@ -0,0 +1,375 @@
+
+const dataWithIcons = dataWithGroups.map(item => ({
+  ...item,
+  group: undefined,
+}));
+
+const dataWithLabels = dataWithIcons.map(item => ({
+  ...item,
+  icon: undefined,
+}));
+
+const args = {
+  data: dataWithLabels,
+  size: "md",
+  label: "Field type",
+  description: undefined,
+  error: undefined,
+  placeholder: "No semantic type",
+  searchable: false,
+  creatable: false,
+  disabled: false,
+  readOnly: false,
+  withAsterisk: false,
+  dropdownPosition: "flip",
+};
+
+const sampleArgs = {
+  value: [dataWithLabels[0].value],
+  description: "Determines how Metabase displays the field",
+  error: "required",
+};
+
+const argTypes = {
+  data: {
+    control: { type: "json" },
+  },
+  size: {
+    options: ["xs", "md"],
+    control: { type: "inline-radio" },
+  },
+  label: {
+    control: { type: "text" },
+  },
+  description: {
+    control: { type: "text" },
+  },
+  error: {
+    control: { type: "text" },
+  },
+  placeholder: {
+    control: { type: "text" },
+  },
+  searchable: {
+    control: { type: "boolean" },
+  },
+  creatable: {
+    control: { type: "boolean" },
+  },
+  disabled: {
+    control: { type: "boolean" },
+  },
+  readOnly: {
+    control: { type: "boolean" },
+  },
+  withAsterisk: {
+    control: { type: "boolean" },
+  },
+  dropdownPosition: {
+    options: ["bottom", "top", "flip"],
+    control: { type: "inline-radio" },
+  },
+};
+
+const DefaultTemplate = args => <MultiSelect {...args} />;
+
+const VariantTemplate = args => (
+  <Stack>
+    <MultiSelect {...args} />
+    <MultiSelect {...args} variant="unstyled" />
+  </Stack>
+);
+
+const Default = DefaultTemplate.bind({});
+const EmptyMd = VariantTemplate.bind({});
+const AsteriskMd = VariantTemplate.bind({});
+const ClearableMd = VariantTemplate.bind({});
+const DescriptionMd = VariantTemplate.bind({});
+const DisabledMd = VariantTemplate.bind({});
+const ErrorMd = VariantTemplate.bind({});
+const ReadOnlyMd = VariantTemplate.bind({});
+const IconsMd = VariantTemplate.bind({});
+const GroupsMd = VariantTemplate.bind({});
+const LargeSetsMd = VariantTemplate.bind({});
+const SearchableMd = VariantTemplate.bind({});
+const CreatableMd = VariantTemplate.bind({});
+const EmptyXs = VariantTemplate.bind({});
+const AsteriskXs = VariantTemplate.bind({});
+const ClearableXs = VariantTemplate.bind({});
+const DescriptionXs = VariantTemplate.bind({});
+const DisabledXs = VariantTemplate.bind({});
+const ErrorXs = VariantTemplate.bind({});
+const ReadOnlyXs = VariantTemplate.bind({});
+const IconsXs = VariantTemplate.bind({});
+const GroupsXs = VariantTemplate.bind({});
+const LargeSetsXs = VariantTemplate.bind({});
+const SearchableXs = VariantTemplate.bind({});
+const CreatableXs = VariantTemplate.bind({});
+
+export default {
+  title: "Inputs/MultiSelect",
+  component: MultiSelect,
+  args,
+  argTypes,
+};
+
+export const Default_ = {
+  render: Default,
+  name: "Default",
+};
+
+export const EmptyMd_ = {
```

Comment:
Probably this indirection is no longer needed, and you could have `EmptyMd` without `_` and set `VariantTemplate` in `render`

Reply:
should be fixed in f039d27be2c98c2d4b55cc3dd98e65679bd8c9c3